### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Update for Capacitor 6. 

I tested SPM support but ran into this issue with a hardcoded ios 13 platform version. I created a feature request to make it configurable - https://github.com/ionic-team/capacitor/issues/7544

Modifying the template I was able to verify the plugin works with SPM and Pods

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            name: "UaCapacitorAirshipPlugin",
+            dependencies: [
+                .product(name: "Capacitor", package: "capacitor-swift-pm"),
+                .product(name: "Cordova", package: "capacitor-swift-pm"),
+                .product(name: "AirshipFrameworkProxy", package: "airship-mobile-framework-proxy")
+            ],
+            path: "ios/Plugin"
+        ),
+        .target(
+            name: "UaCapacitorAirship",
```
Comment (rlepinski) [inline Package.swift:27]: SPM you cant have mixed source targets so I had to break this up into its own file to do the bootloader

File: android/src/main/java/com/airship/capacitor/CapacitorAutopilot.kt [hunk 2]
```diff
@@ -20,15 +20,14 @@ class CapacitorAutopilot : BaseAutopilot() {
...
-        airship.analytics.registerSDKExtension(Analytics.EXTENSION_CORDOVA, AirshipCapacitorVersion.version)
+        airship.analytics.registerSDKExtension(Extension.CAPACITOR, AirshipCapacitorVersion.version)
     }
 
     override fun createConfigBuilder(context: Context): AirshipConfigOptions.Builder {
         val pluginConfig = CapConfig.loadDefault(context).getPluginConfiguration("Airship")
         val proxyConfig = ProxyConfig(JsonValue.wrapOpt(pluginConfig.getObject("config")).optMap())
-        val builder =  AirshipConfigOptions.newBuilder().applyDefaultProperties(context)
+        val builder =  AirshipConfigOptions.newBuilder()
+        try { builder.tryApplyDefaultProperties(context) } catch (_: Exception) {}
```
Comment (rlepinski) [inline android/src/main/java/com/airship/capacitor/CapacitorAutopilot.kt:30]: Prevents logging that it cant find the config

File: ios/Bootloader/AirshipCapacitorBootstrap.m [hunk 3]
```diff
@@ -4,9 +4,10 @@
 
 #if __has_include(<UACapacitorAirship/UACapacitorAirship-Swift.h>)
 #import <UACapacitorAirship/UACapacitorAirship-Swift.h>
+#elif __has_include("UACapacitorAirship-Swift.h")
+#import "UaCapacitorAirship-Swift.h"
 #else
-#import <Capacitor/
-#import "UACapacitorAirship-Swift.h"
+@import UaCapacitorAirshipPlugin;
```
Comment (rlepinski) [inline ios/Bootloader/AirshipCapacitorBootstrap.m:10]: SPM fallback

File: android/src/main/java/com/airship/capacitor/AirshipPlugin.kt [hunk 4]
```diff
@@ -89,168 +89,171 @@ class AirshipPlugin : Plugin() {
...
-            }
-            "push#clearNotification" -> call.resolveResult(method) { proxy.push.clearNotification(arg.requireString()) }
-            "push#clearNotifications" -> call.resolveResult(method) { proxy.push.clearNotifications() }
-            "push#getPushToken" -> call.resolveResult(method) { proxy.push.getRegistrationToken() }
-            "push#android#isNotificationChannelEnabled" -> call.resolveResult(method) {
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-                    proxy.push.isNotificationChannelEnabled(arg.requireString())
-                } else {
-                    true
+        scope.launch {
```
Comment (rlepinski) [inline android/src/main/java/com/airship/capacitor/AirshipPlugin.kt:92]: Same as before, I just moved this to be launched in the scope so its called on the main thread

**Review by crow — APPROVED** [review]

**Review by jyaganeh — APPROVED** [review]

**Pull request merged**